Source/msg.cpp

```diff
@@ -804,15 +797,15 @@ void DeltaLoadLevel()
 					items[ii]._iAC = sgLevels[currlevel].item[i].bAC;
 					items[ii].dwBuff = sgLevels[currlevel].item[i].dwBuff;
 				}
-				x = sgLevels[currlevel].item[i].x;
-				y = sgLevels[currlevel].item[i].y;
+				int x = sgLevels[currlevel].item[i].x;
+				int y = sgLevels[currlevel].item[i].y;
 				if (!CanPut({ x, y })) {
-					done = false;
-					for (k = 1; k < 50 && !done; k++) {
-						for (j = -k; j <= k && !done; j++) {
-							yy = y + j;
-							for (l = -k; l <= k && !done; l++) {
-								xx = x + l;
+					bool done = false;
+					for (int k = 1; k < 50 && !done; k++) {
+						for (int j = -k; j <= k && !done; j++) {
+							int yy = y + j;
+							for (int l = -k; l <= k && !done; l++) {
+								int xx = x + l;

```

Comment:
My god...

Reply:
he's not even yours! 😃

Reply:
This might be the most indented code I've ever seem in a real code actually... now that I think about it.

Reply:
Still two less then what I have to deal with professionally.